File: 0075-sort-colors/0075-sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int h[3] ={0}; // frequencies of i = 0,1,2
        for(int i : nums) h[i]++;
        int j = 0;
        for(int i = 0; i<=2; i++){
            while(h[i]!=0){
                nums[j] = i;
                h[i]--;
                j++;
            }
        }
    }
};

File: 0075-sort-colors/0075-sort-colors.js
/**
 * @param {number[]} nums
 * @return {void} Do not return anything, modify nums in-place instead.
 */
var sortColors = function(nums) {
    const freq = new Array(3).fill(0);
    nums.forEach(i => {
        freq[i]++;
    })
    let j = 0;
    for(let i = 0; i<=2; i++){
        while(freq[i]!=0){
            nums[j++] = i;
            freq[i]--;
        }
    }
};